raiden/transfer/node.py

```diff
@@ -773,10 +783,16 @@ def handle_state_change(chain_state: ChainState, state_change: StateChange) -> T
             state_change,
         )
     elif type(state_change) == ActionUpdateTransportSyncToken:
+        # TODO: deprecated, remove on next db version bump
```

Comment:
We need to bump the DB version anyway for adding `ActionUpdateTransportAuthData`, so you could remove it right away.

Reply:
@ulope as there's no `ActionUpdateTransportAuthData` event in the database, I don't think we need to bump db yet, as only new events will be generated. OTOH, that's why I kept the `ActionUpdateTransportSyncToken` state change, although kind of noop right now.

Reply:
Yes we do, old client versions can't process the `ActionUpdateTransportAuthData` event so we must not write it into a database version that could be read by those.

Reply:
Ok, will remove and bumb db version then